hack/common.sh

```diff
@@ -305,6 +305,7 @@ os::build::internal::build_binaries() {
         eval "platform_goflags=(${!platform_goflags_envvar:-})"
 
         GOOS=${platform%/*} GOARCH=${platform##*/} go install \
+          -pkgdir ${GOPATH}/src/${OS_GO_PACKAGE} \
```

Comment:
@ncdc Correct.  And if we don't have that there, then everything built tries to be put in the go global area (/usr/lib/golang/pkg/linux_386/).  This works if you are a privileged user, but doesn't if you are not.